### DISCUSSION TITLE
Create API for privacy.com.br

### DISCUSSION THREAD

**Discussion by hammerheaddf** [DIGITALCRIMINALS/OnlyFans#308] [Ideas]
(feature request)
[privacy.com.br](https://privacy.com.br) is like a Brazilian OnlyFans, with lots of interesting creators, and sometimes cheaper than OF. I'd volunteer to create an API for it, but I guess this task is way beyond my Python skillset. Therefore I'm humbly asking the community to have a look into it.

**Comment by hammerheaddf** [comment]
So, expanding on the request, could anybody give us the skinny on writing an API for a different website?

**Reply by jeffersonlsz** [reply]
I am working on a extension for privacy website. If you want you can send me a private message so I can show you what I've made so far....

**Reply by hammerheaddf** [reply]
Hey, @jeffersonlsz, are you still working on it? I've got some free time now and I'm about to give it a try.
